title-2-conservation/crp/crp_state_distribution_data.json

```diff
@@ -0,0 +1,4354 @@
+{
+  "2018-2022": [
+    {
+      "state": "KS",
+      "programs": [
+        {
+          "programName": "Total General Sign-Up",
+          "totalContracts": 92885,
+          "totalFarms": 64036,
+          "totalAcre": 7317993,
+          "totalRent1K": 301096,
```

Comment:
It looks like in the CSV, this value denotes value in thousand dollars. I think it will be better to convert this into dollars so that we can do conversions to millions or billions from existing frontend logic. 

Also, in that case, it will be good to change the variable name to rentalPaymentsInDollars or something similar.

Reply:
It is just totalRent1k right? Contract or Farms are not dollars but just he number of occasion, right?

Reply:
I changed the title to totalRentInDollars and value by multiplying by 1000.

Reply:
Thanks.